piperider_cli/profiler/schema.json

```diff
@@ -443,6 +449,79 @@
         }
       }
     },
+    "metric": {
+      "title": "Metric",
```

Comment:
If possible, I think it's good to be more specific here in terms of what "metric" we're talking about. 
B/C we already have profile metrics, table metrics, etc (it's a generic word that exists), 

Yet we are using it in a specific way here, in terms of DBT.

Suggestion: rename as `DbtMetric` or `BusinessMetric` to distinguish from other metric concepts.

Reply:
Agree, we have mentioned `metric` before. It's better to have a specific name.
I prefer `BusinessMetric` because it will not be limited to dbt integration.